lua/zipkin.lua

```diff
@@ -90,6 +90,11 @@ function zipkin.emit_syslog(headers, start_time, end_time, response)
             ngx.var.request_uri,
             ngx.var.server_protocol
         )
+        -- For DELETEs which might not have any response,assume spectre_cache_status is MISS 
+        local spectre_cache_status = "MISS"
+        if ngx.var.request_method ~= 'DELETE' then
+          spectre_cache_status = response.headers['Spectre-Cache-Status']
+        end
```

Comment:
For better performance this could be rewritten as:
```
local spectre_cache_status = response.headers['Spectre-Cache-Status']
if ngx.var.request_method == 'DELETE' then
    spectre_cache_status = "MISS"
end
```
Given that most of the cases are `GET` it would save 1 instruction